contracts/lazyclaim/ERC721LazyClaim.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: manifold.xyz
+
+import "@manifoldxyz/creator-core-solidity/contracts/core/IERC721CreatorCore.sol";
+import "@manifoldxyz/libraries-solidity/contracts/access/AdminControl.sol";
+import "@manifoldxyz/creator-core-solidity/contracts/extensions/ICreatorExtensionTokenURI.sol";
+
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import "@openzeppelin/contracts/interfaces/IERC165.sol";
+
+import "./IERC721LazyClaim.sol";
+
+/**
+ * Lazy claim with optional whitelist ERC721 tokens
+ */
+contract ERC721LazyClaim is IERC165, IERC721LazyClaim, ICreatorExtensionTokenURI, ReentrancyGuard {
+  mapping(address => uint) public claimCounts;
```

Comment:
Don't need this, store as part of the claim object.

Reply:
Better to name this as claimIndex

Reply:
Commented all my mappings

Have to push back on naming this `claimIndex` as that name is already used in code to refer to a member of range(0, claimCount)